test/Microsoft.DotNet.ProjectModel.Tests/LockFileReaderTests.cs

```diff
@@ -0,0 +1,54 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.IO;
+using System.Text;
+using FluentAssertions;
+using Microsoft.DotNet.ProjectModel.Graph;
+using Xunit;
+using NuGet.Versioning;
+
+namespace Microsoft.DotNet.ProjectModel.Tests
+{
+    public class LockFileReaderTests
+    {
+        [Theory]
+        [InlineData("", null)]
+        [InlineData(@"""path"": null,", null)]
+        [InlineData(@"""path"": ""foo/1.0.0"",", "foo/1.0.0")]
+        public void AllowsPackageLibraryPath(string pathProperty, string expected)
```

Comment:
What will happen when a user updates their CLI but doesn't re-restore? How is that UX managed? Can we have a test?